### ISSUE TITLE
Which packages to upgrade for rc1

### ISSUE THREAD

**Issue by pierre-weceipt** [dotnet/wcf#541]
Hello guys, I've been using the succesive beta for .net5 for a few months and added around beta6 a project based on WCF service I added with the VSTools for .net45 projects and then copied in a .net5 class project.

It is working fine as long as I keep the following packages and stay in runtime `dnx46` : 

``` json
"dependencies": {
    "System.ServiceModel.Http": "4.0.10-beta-22816",
    "System.ServiceModel.Primitives": "4.0.0-beta-22816"
}
```

The problem is the following : each time I tried to upgrade them to current version (now `beta-23516`) or change the dnx, errors are poping everywhere in my code, mostly in the `Reference.cs` file with `DataContractAttribute`. I suppose I need to add packages and/or change some part of the code but could'nt find a tutorial or example. Do you have any idea on how I could upgrade the packages without breaking anything?

**Comment by hongdai** [comment]
Upgrade between beta version involves several manual steps. 

Please give below steps a try. 
1) upgrade DNX Runtime on the machine, try the below commands:
dnvm update-self
dnvm upgrade
2) Change your project Application and Debug property from beta5 to the current version.
3) Update your porject.json file, you will need to include following new dependencies:
        "System.Net.Security": "4.0.0-beta-*",
        "System.Net.NameResolution": "4.0.0-beta-*",
4) Open a VS2015 deveoloper window. Go to the root of your project and run
dnu restore

Note that I haven't tried the steps myself. But it should work.

**Comment by hongdai** [comment]
Also reference Install ASP.NET 5 from the command-line in the below link to install RC1.
https://docs.asp.net/en/latest/getting-started/installing-on-windows.html

**Comment by pierre-weceipt** [comment]
Ok I was not clear enough : the other projects in my solution are already in rc1 and working fine. The only project with packages that I could not update is the class project containing the WCF service : if I do, everything gets broken.

**Comment by hongdai** [comment]
It's most likely because your package dependencies are not fully updated. Run dnu restore from the root of your project is helpful.

**Comment by pierre-weceipt** [comment]
I had the same problem with beta7 and 8 and already tried the dnu restore (though I did not try it with rc1). It would be nice to have a working sample somewhere.

**Comment by hongdai** [comment]
Upgrade between betas worked for me. We don't currently have a sample. In the mean time, could you tell me what specific errors you are seeing?

**Comment by pierre-weceipt** [comment]
All of the errors come from :
```
System.Runtime.Serialization.DataContractAttribute
System.Runtime.Serialization.DataMemberAttribute
```
Can't find the namespace, I suppose things where moved after `beta-22816`.
IntelliSense proposed me to add `"System.ServiceModel.Primitives": "4.1.0-beta-23516"` but it did not change anything even after `dnu restore`

**Comment by hongdai** [comment]
Those namespaces are from System.Runtime.Serialization.Primitives.dll. Could you try to add it to your project.json file. It should be automatically added by Nuget. 

"System.Runtime.Serialization.Primitives": "4.0.10"

**Comment by pierre-weceipt** [comment]
I tried to add the package in version `4.0.10` and then `4.0.11-beta-23516` but add still the errors. However when I switched to `dnxcore50` and removed `System.CodeDom.Compiler.GeneratedCodeAttribute("System.ServiceModel", "4.0.0.0")]` from the `Reference.cs` file, errors where gone. 
Unfortunatly I can't use `dnxcore50` at the moment because of some dependancy in another project in the solution.
